sys-cluster/teleport/teleport-2.6.0.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit eutils golang-build systemd user
+
+DESCRIPTION="Modern SSH server for teams managing distributed infrastructure"
+HOMEPAGE="https://gravitational.com/teleport"
+
+EGO_PN="github.com/gravitational/${PN}/..."
+
+if [[ ${PV} == "9999" ]] ; then
+	inherit git-r3 golang-vcs
+	EGIT_REPO_URI="https://github.com/gravitational/${PN}.git"
+else
+	inherit golang-vcs-snapshot
+	SRC_URI="https://github.com/gravitational/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm"
+fi
+
+IUSE=""
+LICENSE="Apache-2.0"
+RESTRICT="test strip"
+SLOT="0"
+
+DEPEND="
+	app-arch/zip
+	>=dev-lang/go-1.9.2"
+RDEPEND=""
+
+src_prepare() {
+	default
+
+	sed -i -e 's/-j 3/-j 1/g' src/${EGO_PN%/*}/Makefile || die
+}
+
+src_compile() {
+	BUILDFLAGS="" GOPATH="${S}" emake -j1 -C src/${EGO_PN%/*} full
+}
+
+src_install() {
+	keepdir /var/lib/${PN} /etc/${PN}
+	dobin src/${EGO_PN%/*}/build/{tsh,tctl,teleport}
+
+	insinto /etc/${PN}
+	newins "${FILESDIR}"/${PN}-2.6.yaml ${PN}.yaml
+
+	newinitd "${FILESDIR}"/${PN}.init.d ${PN}
```

Comment:
what are strange extensions? `.conf` and `.init` are correct ones, to just sum up, all the existing ebuilds for the package are having the same mistakes, not only these ones

Reply:
`teleport.yaml` is the configuration file for teleport installed in `/etc/teleport`.

I there a reference for the correct extensions for these files?  The `teleport.init.d` file is being installed correctly under `/etc/init.d/teleport`

Reply:
Is this explanation sufficient?  Or is there documentation somewhere I can follow for these extensions?